Bottom: fb19f003dc1f917d4ad2e32c9b2db10378e75042
Top:    96c7d678475e7240f10e2802ff4cb513df535dc6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-15 17:06:15 +0100

report.c: Return 1 for all failure conditions in ReportNetLengthByName()


---

diff --git a/src/report.c b/src/report.c
index 4d5e27b..cf3635b 100644
--- a/src/report.c
+++ b/src/report.c
@@ -812,7 +812,7 @@ ReportNetLengthByName (char *tofind, int x, int y)
 	  if ((elt_pattern = re_comp (tofind)) != NULL)
 	    {
 	      Message (_("re_comp error: %s\n"), elt_pattern);
-	      return (false);
+	      return (1);
 	    }
 #endif
 	}
